strawberryfields/backends/tfbackend/tests/test_state_preparations.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Unit tests for various state preparation operations"""
```

Comment:
Why are the tests in `test_squeezed_state_preparation.py` just not included in this test file instead?

Reply:
Think it's probably just a function of the order those test files were created

Reply:
lol. Ditto my comment above, this is another thing to keep in mind for a later test refactor PR.